src/IKEA/Tradfri/Traits/ProvidesBrightness.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+declare(strict_types=1);
+
+namespace IKEA\Tradfri\Traits;
+
+/**
+ * Trait ProvidesBrightness.
+ */
+trait ProvidesBrightness
+{
+    /**
+     * @var float
+     */
+    protected $_brightness;
+
+    /**
+     * Get Brightness.
+     *
+     * @return float
+     */
+    public function getBrightness(): float
+    {
+        return (float) $this->_brightness;
+    }
+
+    /**
+     * Set Brightness.
+     *
+     * @param int $brightness
+     *
+     * @return static
+     */
+    public function setBrightness(int $brightness): self
+    {
+        if ($brightness < 0) {
+            $this->_brightness = 0;
+        } else {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method setBrightness uses an else expression. Else is never necessary and you can simplify the code to work without else.](https://app.codacy.com/manual/Fahl-Design/ikea-tradfri-php/pullRequest?prid=2789116)